activity.py

```diff
@@ -25,7 +25,7 @@
 import os
 import gi
 gi.require_version('Gtk', '3.0')
-from gi.repository import Gtk
+from gi.repository import Gtk , GObject
```

Comment:
Changed line does not pass flake8, E203.

Reply:
Ok will correct it. Thanks